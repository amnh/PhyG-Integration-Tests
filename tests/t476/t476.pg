--softwire tests, W15, netdel, atrandom, steepest
set(seed:1634561640)
set(outgroup:T0)
set(graphtype:softwired)
set(graphfactor:w15) 
read(fasta:"Mut2_1_sars-cov_reduced.fasta")
read(fasta:"Mut2_2_sars-cov_reduced.fasta")
read(newick:"../t467/t467.tre")
refine(netdel, atrandom, steepest)
report("t476_ia.ss", ia, overwrite)
report("t476.csv", data, overwrite)
report("t476.tre", graphs, newick, overwrite)
report("t476_gv.tre", dotpdf, graphs, overwrite)