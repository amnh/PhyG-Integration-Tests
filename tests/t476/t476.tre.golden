(T0:0.0,((T17:72.0,(T21:97.0,(T11:86.0,((T16:41.0,(T1:0.0)#HTU32:0.0)HTU31:33.0,(T4:0.0)#HTU33:0.0)HTU30:47.0)HTU29:90.0)HTU28:56.0)HTU27:88.0,((T6:84.0,(T18:85.0,((T8:81.0,(T2:68.0,(T9:72.0,(T7:66.0,((T19:66.0,(T10:70.0,(T20:95.0,#HTU32:0.0)HTU45:36.0)HTU44:84.0)HTU43:82.0,(T15:89.0,T24:74.0)HTU46:70.0)HTU42:63.0)HTU41:76.0)HTU40:72.0)HTU39:70.0)HTU38:69.0,((T14:78.0,(T13:87.0,T5:76.0)HTU49:70.0)HTU48:83.0,(T22:90.0,T23:68.0)HTU50:76.0)HTU47:87.0)HTU37:53.0)HTU36:74.0)HTU35:75.0,((T12:74.0,T3:88.0)HTU52:83.0,#HTU33:0.0)HTU51:26.0)HTU34:71.0)HTU26:1.0)HTU25[4512.0625];
(T0:0.0,(((T6:84.0,(T18:85.0,((T8:81.0,(T2:68.0,(T9:72.0,(T7:66.0,((T19:66.0,(T10:70.0,(T20:95.0,(T1:0.0)#HTU39:0.0)HTU38:36.0)HTU37:84.0)HTU36:82.0,(T15:89.0,T24:74.0)HTU40:70.0)HTU35:63.0)HTU34:76.0)HTU33:72.0)HTU32:70.0)HTU31:69.0,((T14:78.0,(T13:87.0,T5:76.0)HTU43:70.0)HTU42:83.0,(T22:90.0,T23:68.0)HTU44:76.0)HTU41:87.0)HTU30:53.0)HTU29:74.0)HTU28:75.0,((T12:74.0,T3:88.0)HTU46:83.0,(T4:0.0)#HTU47:0.0)HTU45:26.0)HTU27:70.0,(T17:72.0,(T21:96.0,(T11:86.0,((T16:77.0,#HTU39:0.0)HTU52:33.0,#HTU47:0.0)HTU51:47.0)HTU50:90.0)HTU49:56.0)HTU48:87.0)HTU26:1.0)HTU25[4512.0625];
